pact.cabal

```diff
@@ -1,6 +1,5 @@
-
 name:                pact
-version:             2.6.1
+version:             2.6.1.1
```

Comment:
What's our policy for bumping Pact's version numbers?

Reply:
No, I forgot to edit the nix file -- thanks.  I don't know what our policy is, I was just giving it the most minor bump for now.

Reply:
We've only been bumping on public release, so unless you need this, remove it.

Reply:
ok, done